assets/pages/oneoff-contributions/components/formFields.jsx

```diff
@@ -3,18 +3,97 @@
 // ----- Imports ----- //
 
 import React from 'react';
-import NameFormField from './nameFormField';
-import EmailFormFieldContainer from './emailFormFieldContainer';
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+import TextInput from 'components/textInput/textInput';
+import {
+  setFullName,
+  setEmail,
+  type Action as UserAction,
+} from 'helpers/user/userActions';
+import { type UserFormFieldAttribute, shouldShowError } from 'helpers/checkoutForm/checkoutForm';
+import {
+  type Action as CheckoutAction,
+  setFullNameShouldValidate,
+  setEmailShouldValidate,
+} from './contributionsCheckoutContainer/checkoutFormActions';
+import { formFields } from '../helpers/checkoutFormFields';
+import { type PageState as State } from '../oneOffContributionsReducer';
 
 
+// ----- Types ----- //
+
+type PropTypes = {
+  fullName: UserFormFieldAttribute,
+  email: UserFormFieldAttribute,
+  setFullName: (string) => void,
+  setEmail: (string) => void,
+  setFullNameShouldValidate: () => void,
+  setEmailShouldValidate: () => void,
+};
+
+// ----- Map State/Props ----- //
+
+function mapStateToProps(state: State) {
+
+  const { fullName, email } = formFields(state);
```

Comment:
This is a [selector](https://redux.js.org/recipes/computingderiveddata), could it be given a name like `getFormFields` to make it more obvious that that's what it's doing?